apps/mull-ui/src/app/pages/register/register.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import logo from '../../../assets/mull-logo.png';
+import './register.scss';
+
+/* eslint-disable-next-line */
+export interface RegisterProps {
+  history: History;
+}
+
+export const Register = ({ history }) => {
```

Comment:
No explicit export needed here. There's already an export default at the bottom of the file.

Reply:
Missing prop declaration

Reply:
Was wondering which prop declaration that you're talking about?

Reply:
He means that you're not declaring what type your props are
```suggestion
export const Register = ({ history }: RegisterProps) => {
```